CompatBot/Commands/Misc.cs

```diff
@@ -80,7 +84,7 @@ internal sealed class Misc: BaseCommandModuleCustom
             "Boooooooo!", "Poopy", "Smelly", "Feeling-breaker", "Annoying",
             "Boring", "Easily forgettable", "An Abomination", "A Monstrosity", "Truly horrific",
             "Filled with despair!", "Eroded by despair", "Hopeless…", "It's pretty foolish to want to rate this", "Cursed with misfortune",
-            "Nothing but terror",
+            "Nothing but terror", "Not good, at all", "Please uninstall",
```

Comment:
“Rate this comic book”
“Please uninstall”
“Um what”

Reply:
Yeah, I think i got the wrong idea of what it should do, at first.
But now I think I fixed what you requested